javascript/javascript1/week3/traveltime.js

```diff
@@ -0,0 +1,11 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function totalTime(travelInformation) {
+  const time = (travelInformation.destinationDistance) / (travelInformation.speed);
+  return `${Math.floor(time)} hours and ${Math.floor((time-Math.floor(time)) * 60)} minutes`;
```

Comment:
`Math.floor((time-Math.floor(time)) * 60)` can be simplified.

Reply:
Tried but not sure you meant like this.